pkg/client/unversioned/testclient/simple/simple_testclient.go

```diff
@@ -53,6 +54,7 @@ type Response struct {
 
 type Client struct {
 	*client.Client
+	*generated.TestgroupClient
```

Comment:
It would make me accept it if there's really no other way, but I'd still prefer to not have to do this.